### ISSUE TITLE
Command line encoding error

### ISSUE THREAD

**Issue by jerethi** [staxrip/staxrip#862]
**Describe the bug**
Command Line audio encoding errors out, even with default settings

**Expected behavior**
Audio should encode without error

**How to reproduce the issue**

Using the default command line audio encoding settings causes an error.

**Provide information**

Here is the error that displays:

![image](https://user-images.githubusercontent.com/47827013/163695949-bcbcd79e-4399-4b19-acac-75a4fe9cc641.png)

Using StaxRip version: 2.11

**Comment by jerethi** [comment]
Additional information: I opened up command prompt and ran some tests with the included version of ffmpeg. Can confirm that all ran smoothly and correctly when working in a command terminal. So, there appears to be an issue in how Staxrip is passing commands via the GUI, it would seem.

**Comment by jerethi** [comment]
Update: I believe I discovered the issue. At least in my installation, the default settings for the Command line audio encoding has quotations around %input and %output, which is what is returning the error. After deleting those quotes, the arguments pass correctly. Again, I tried this on a fresh install of Staxrip and the issue was present - not sure if the configuration files from my normal installation were somehow copied over to the fresh installation which caused the issue. In any event, I'm working properly now.
